app/controllers/uploads_controller.rb

```diff
@@ -57,7 +57,9 @@ def new_upload(csv_type)
     upload = Upload.new(csv_type: csv_type)
     upload.skip_lines = defaults(csv_type)['skip_lines']
     upload.col_sep = defaults(csv_type)['col_sep']
-
+    upload.force_simple_split = defaults(csv_type)['force_simple_split']
+    upload.strip_chars_from_headers = defaults(csv_type)['strip_chars_from_headers']
+    # end
```

Comment:
Looks like you accidentally left in a comment.

Reply:
So I have. It's gone!